pom.xml

```diff
@@ -278,11 +278,6 @@
             <artifactId>log4j-api</artifactId>
             <version>${log4j-version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-1.2-api</artifactId>
-            <version>${log4j-version}</version>
-        </dependency>
```

Comment:
Do we need to add any exclusions?

Reply:
oh i see double check no log4j1. Yes got it. Have confirmed no log4j1